src/EntryPoint/ReferenceEntryPoint.php

```diff
@@ -213,8 +213,9 @@ public function bankDetails(
         $identifierValue
     ) {
         $response = $this->request(
-            'GET',
+            'POST',
             'reference/bank_details',
```

Comment:
@korki the find endpoint for bank_details was actually changed to `reference/bank_details/find` so you will need to update this here.

See

https://github.com/CurrencyCloud/currencycloud-swagger/pull/247/files#diff-503fd62126a95472639899ef0c06546ed50daa650c1e5cde73cd3ba04f8e85acR12852